tests/MongoDB.Driver.Core.Tests/Core/Operations/RetryabilityHelperTests.cs

```diff
@@ -118,18 +170,18 @@ public void IsRetryableReadException_should_return_expected_result_using_code(in
         }
 
         [Theory]
-        [InlineData(typeof(IOException), false)]
-        [InlineData(typeof(MongoCursorNotFoundException), false)]
-        [InlineData(typeof(MongoConnectionException), true)]
-        [InlineData(typeof(MongoNodeIsRecoveringException), true)]
-        [InlineData(typeof(MongoNotPrimaryException), true)]
-        public void IsRetryableWriteException_should_return_expected_result_using_exception_type(Type exceptionType, bool expectedResult)
+        [ParameterAttributeData]
+        public void IsRetryableWriteException_should_work_as_expected([Values(false, true)] bool hasRetryableWriteLabel)
```

Comment:
We usually say `_should_return_expected_result` instead of `_should_work_as_expected`:

    public void IsRetryableWriteException_should_return_expected_result([Values(false, true)] bool hasRetryableWriteLabel)

Reply:
done